web/tests/compose_closed_ui.test.cjs

```diff
@@ -31,6 +31,10 @@ const compose_closed_ui = zrequire("compose_closed_ui");
 const {Filter} = zrequire("filter");
 const {MessageList} = zrequire("message_list");
 const {MessageListData} = zrequire("message_list_data");
+const {set_realm} = zrequire("state_data");
+
+const REALM_EMPTY_TOPIC_DISPLAY_NAME = "test general chat";
```

Comment:
I think it's OK to write tests with this being "general chat", since we expect that to be its final value.

Reply:
Changed it to "general chat"